src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateCreationSettings.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace WcfTestBridgeCommon
+{
+    [Serializable]
+    public class CertificateCreationSettings
+    {
+        public CertificateCreationSettings()
+        {
+            Subjects = new string[] { string.Empty };
+            IsValidCert = true;
+        }
```

Comment:
I'd be careful of using C# 6 yet.  We don't know the state of Roslyn and building on Linux yet.